content/en/administrators_guide/build.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Build your Datadog installation
+description: Setup and install Datadog as an administrator to avoid pit-falls down the road
```

Comment:
```suggestion
description: Setup and install Datadog as an administrator to avoid pitfalls down the road
```

Reply:
I feel that each page should have a different description, but that can be done in a future update.